nalgebra-sparse/src/csr.rs

```diff
@@ -33,8 +34,9 @@ use std::slice::{Iter, IterMut};
 /// // change the sparsity pattern of the matrix after it has been constructed. The easiest
 /// // way to construct a CSR matrix is to first incrementally construct a COO matrix,
 /// // and then convert it to CSR.
-/// # use nalgebra_sparse::coo::CooMatrix;
-/// # let coo = CooMatrix::<f64>::new(3, 3);
+///
+/// let coo = CooMatrix::<f64>::new(3, 3);
+/// coo.push(2, 0, 1.0);
```

Comment:
Ah, didn't catch this upon review, but thankfully the CI did. Need to make `coo` mutable if you want to push something to it!